bindings/java/src/test/java/org/ucx/jucx/UcpListenerTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) Mellanox Technologies Ltd. 2001-2019.  ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+package org.ucx.jucx;
+
+import org.junit.Test;
+import org.ucx.jucx.ucp.*;
+
+import java.io.IOException;
+import java.net.InetAddress;
+import java.net.InetSocketAddress;
+import java.net.NetworkInterface;
+import java.net.SocketException;
+import java.nio.ByteBuffer;
+import java.util.Enumeration;
+import java.util.LinkedList;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class UcpListenerTest {
+    private static final int port = 55331;
```

Comment:
we should be able to get port number as user arg. otherwise it can't run 2 instances in parallel on same Jenkins machine